### ISSUE TITLE
Fix link color contrast accessibility violation

### ISSUE THREAD

**Issue by cdmo** [psu-libraries/psulib_blacklight#591]
link color in announcement bar does not pass wcag aa, it is 3.15:1 at the moment

**Comment by cdmo** [comment]
just marking the approval here, Ruth mentioned in Slack
